manifests/params.pp

```diff
@@ -18,14 +18,20 @@
 
   case $::osfamily {
     'Debian': {
+      $jenkins_user       = 'jenkins'
```

Comment:
Since all the values for `$jenkins_user` and `$jenkins_group` are the same, they don't need to be in the case statement at all.